assets/scss/08-editor/_tip.scss

```diff
@@ -0,0 +1,70 @@
+.tip {
+    @include content-block-vertical-offset();
+    @include content-block-wide();
+
+    padding: spacing(1) $container-padding;
+    border-left: $border-width-thick solid map-get($colors-ui, warning);
+    background-color: transparentize(map-get($colors-ui, warning), 0.75);
+
+    @include breakpoint-up(sm) {
+        padding-right: $container-padding-sm;
+        padding-left: $container-padding-sm;
+    }
+
+    @include breakpoint-up(md) {
+        border-radius: $border-radius;
+    }
+}
+
+.tip h4:first-child,
+.tip h5:first-child,
+.tip h6:first-child {
+    margin-top: 0;
+}
+
+.tip p:last-child {
+    margin-bottom: 0;
+}
+
+@media (prefers-color-scheme: dark) {
+    .tip a,
+    .tip strong,
+    .tip h2,
+    .tip h3,
+    .tip h4,
+    .tip h5,
+    .tip h6 {
+        color: transparentize($color-white, 0.08);
+    }
+
+    .tip h2 a,
+    .tip h3 a,
+    .tip h4 a,
+    .tip h5 a,
+    .tip h6 a {
+        &:hover,
+        &:focus {
+            color: transparentize($color-white, 0.08);
```

Comment:
Jsem zvyklá, že v případě ITCSS, pokud se mi v komponentě opakuje víc jak jednou nějaká CSS vlastnost, tak si její hodnotu předávám do proměnné v `settings/` a posléze použiji u dané komponenty. Tak se raději zeptám, je tenhle přístup správný? 🙂

Reply:
Nejsem tak striktní, ale ano, dává to smysl. Udělal jsem z toho „privátní“ proměnnou.